sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultBackupClient.cs

```diff
@@ -444,6 +536,87 @@ internal virtual Response<FullBackupDetailsInternal> GetBackupDetails(string job
             }
         }
 
+        /// <summary>
+        /// Initiates a pre-backup check on the Key Vault. This operation checks if it is possible to back up the entire collection of keys from a Key Vault.
+        /// </summary>
+        /// <param name="blobStorageUri">The <see cref="Uri"/> for the blob storage resource.</param>
+        /// <param name="sasToken">Optional Shared Access Signature (SAS) token to authorize access to the blob. If null, Managed Identity will be used to authenticate instead.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="blobStorageUri"/> or <paramref name="sasToken"/> is null.</exception>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        /// <returns>A <see cref="KeyVaultBackupOperation"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<KeyVaultBackupOperation> StartPreBackupAsync(Uri blobStorageUri, string sasToken = default, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNull(blobStorageUri, nameof(blobStorageUri));
+            Argument.AssertNotNull(sasToken, nameof(sasToken));
+
+            using DiagnosticScope scope = _diagnostics.CreateScope($"{nameof(KeyVaultBackupClient)}.{nameof(StartPreBackup)}");
+            scope.Start();
+            try
+            {
+                Operation<FullBackupDetailsInternal> operation = await _restClient.PreFullBackupAsync(
+                    WaitUntil.Started,
+                    new PreBackupOperationParameters(
+                        blobStorageUri.AbsoluteUri,
+                        sasToken,
+                        useManagedIdentity: sasToken == null
```

Comment:
```suggestion
                        useManagedIdentity: (sasToken == default)
```

nit: We initialize to `default` so we should be consistent in checking against it.